tests/conftest.py

```diff
@@ -97,6 +97,16 @@ def boto_session(request):
         return boto3.Session(region_name=DEFAULT_REGION)
 
 
+@pytest.fixture(scope="session")
+def account(boto_session):
+    return boto_session.client("sts").get_caller_identity()["Account"]
+
+
```

Comment:
Do we need to create these two fixtures?

Can't we just use `boto_session` to access `account` and `region` ?

Any benefit for creating this one?

Reply:
For `account`, it's to reduce code duplication. While it's true we could use `boto_session.client("sts").get_caller_identity()["Account"]` every time we want to access the account ID, IMO setting up a fixture for the account is a cleaner implementation.

With respect to the `region` fixture, there's less of a case for that. I subjectively feel it simplifies the implementation, but I also see the argument for why it may be unnecessary. If you feel that we should get rid of it, I'd be okay with doing so.